source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -143,25 +143,25 @@ Related topics: [hierarchies][hierarchy].
 
 The value of `hierarchy` must be an array of hashes.
 
-1. Begin each level of the hierarchy with:
+1.  Begin each level of the hierarchy with:
 
-	* Two spaces of indentation.
-	* A hyphen (`-`).
-	* Another space after the hyphen.
-	* The first key of that level’s hash.
+    -   Two spaces of indentation.
+    -   A hyphen (`-`).
+    -   Another space after the hyphen.
+    -   The first key of that level's hash.
 
-2. Indent the rest of the hash’s keys by four spaces, so they line up with the first key.
+2.  Indent the rest of the hash's keys by four spaces, so they line up with the first key.
 
-3. Put an empty line between hashes, to visually distinguish them.
+3.  Put an empty line between hashes, to visually distinguish them.
 
-   ```
-   hierarchy:
-     - name: "Per-node data"
-       path: "nodes/%{trusted.certname}.yaml"
+    ``` yaml
+    hierarchy:
+      - name: "Per-node data"
+        path: "nodes/%{trusted.certname}.yaml"
 
-     - name: "Per-datacenter business group data"
-       path: "location/%{facts.whereami}/%{facts.group}.yaml"
-     ```
+      - name: "Per-datacenter business group data"
+        path: "location/%{facts.whereami}/%{facts.group}.yaml"
+    ```
```

Comment:
The indentation of this code block was inconsistent. The opening and closing code fences were indented three and five spaces, respectively. They now use consistent four-character indents.